src/Router.cpp

```diff
@@ -64,15 +64,14 @@ void Router::routeRequest(HTTPRequest &request, HTTPResponse &response)
 	if (root.empty())
 		root = "var/";
 	request.setRoot(root);
-	adaptPathForFirefox(request);
 
-	std::cout << YELLOW << "requestTarget: " << request.getRequestTarget() << RESET << std::endl;
-	std::cout << GREEN << "Routing request to path: " << request.getPath() << RESET << std::endl;
+	adaptPathForFirefox(request);
+	
+	Debug::log("Routing Request: path = " + request.getPath(), Debug::NORMAL);
 
 	PathValidation pathResult = pathIsValid(response, request);
-	std::cout << BLUE << "path: " << request.getPath() << RESET << std::endl;
-	std::cout << BLUE << "PathValidation: " << pathResult << RESET << std::endl;
-	
+	Debug::log("Routing Request: pathResult = " + toString(pathResult), Debug::NORMAL);
+	Debug::log("Path requested: " + request.getPath(), Debug::NORMAL);
 	// check if method is allowed
```

Comment:
Should this be Debug::NORMAL or should we have an enum ROUTER?

Reply:
Of course ROUTER